packages/core/src/actors.ts

```diff
@@ -295,12 +191,6 @@ export function createPromiseBehavior<T, TEvent extends EventObject>(
           resolvedPromise.then(
             (response) => {
               if (!canceled) {
-                parent?.send(
-                  toSCXMLEvent(doneInvoke(actorContext.name, response) as any, {
-                    origin: actorContext.self
-                  })
-                );
-
                 observers.forEach((observer) => {
                   observer.next?.(response);
                   observer.complete?.();
```

Comment:
In response to an event an observer might get unsubscribed and we shouldn't call `complete` then
```suggestion
                  if (observers.has(observer)) observer.complete?.();
```

Reply:
Nice catch on the edge-case!